app/assets/scss/application.scss

```diff
@@ -158,6 +158,10 @@ $path: "/static/images/";
   }
 }
 
+.break-link {
+  word-break: break-all;
```

Comment:
Too late now, but I have a suspicion that `break-word` with a fallback to `break-all` might be more elegant.